contrib/gin-gonic/gin/option.go

```diff
@@ -73,6 +75,14 @@ func WithResourceNamer(namer func(c *gin.Context) string) Option {
 	}
 }
 
+// WithIgnoreRequest holds the function to use for determining if the
```

Comment:
```suggestion
// WithIgnoreRequest specifies a function to use for determining if the
```
I think that should be more correct. It doesn't "hold" anything, which were the case if it were a variable or a constant. It "does" something.